python/torch_mlir/tools/import_onnx/__main__.py

```diff
@@ -75,6 +75,7 @@ def load_onnx_model(args: argparse.Namespace) -> onnx.ModelProto:
 
     # Load the model, with possible external data coming from the default
     # location, or the location specified on the conmand line.
+    data_prop = False if args.no_data_prop else True
```

Comment:
`data_prop = not args.no_data_prop`. Though you really should just name it `data_prop` and change the default

Reply:
Yeah, that makes sense.